itests/eth_deploy_test.go

```diff
@@ -28,7 +28,9 @@ func TestDeployment(t *testing.T) {
 	// He who writes the second test, shall do that.
 	// kit.QuietMiningLogs()
 
-	blockTime := 100 * time.Millisecond
+	// reasonable blocktime so that the tx sits in the mpool for a bit during the test.
+	// although this is non-deterministic...
+	blockTime := 1 * time.Second
```

Comment:
I'd consider increasing this -- this test isn't really constrained by epochs (needs ~3 epochs), so even if the blocktime were like 5 seconds, it would still finish pretty quickly

Reply:
Plus the boot up time for the chain (around 10 epochs).